src/main/java/com/futurecollars/Task2/Calculator.java

```diff
@@ -0,0 +1,46 @@
+package com.futurecollars.Task2;
+
+public class Calculator {
+
+    private static int counter;
```

Comment:
Do czego nam ten counter?

Reply:
Jest to kalkulator robiony wcześniej na wykładzie przez Rafała i udostępniony. W zasadzie program działa poprawnie identycznie bez niego.